src/batRandom.mliv

```diff
@@ -206,6 +206,15 @@ module State : sig
 ##V>=4.14##val bits32 : t -> Int32.t
 ##V>=4.14##val bits64 : t -> Int64.t
 ##V>=4.14##val nativebits : t -> Nativeint.t
+
+##V>=5##val split : t -> t
+##V>=5##(** Draw a fresh PRNG state from the given PRNG state.
+##V>=5##    (The given PRNG state is modified.)
+##V>=5##    The new PRNG is statistically independent from the given PRNG.
+##V>=5##    Data can be drawn from both PRNGs, in any order, without risk of
+##V>=5##    correlation.  Both PRNGs can be split later, arbitrarily many times.
+##V>=5##    @since 5.0.0 *)
```

Comment:
Minor nitpick: I think other `@since` tags are explicit about whether they talk about Batteries or OCaml version. Same for other OCaml 5 added functions.

Reply:
I also thought about it. I might update those tags on Monday.

Reply:
this has been dealt with